src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/model/ChipsKafkaMessage.java

```diff
@@ -4,7 +4,6 @@
 
     @Override
     public String toString() {
-        //stuff
         return "hello";
```

Comment:
i dont believe we are using this method, should we just remove the override to not confuse things later?